server/controllers/savedJobsController.js

```diff
@@ -48,10 +48,7 @@ exports.saveJob = async (req, res) => {
 };
 
 exports.getSavedJobs = (req, res) => {
-// Kevin and Duncan will add the userID. in the meantime, use hard coded:
-  req.user = {
-    id: 1,
-  };
+  console.log('user id', req.user.id)
```

Comment:
Please remove console log, and instead insert a statement here that will send a 401 back to the user if they are not logged in.  See the example in Slack.